app/views/shared/_nav.html.haml

```diff
@@ -1,25 +1,5 @@
-.navbar.navbar-static-top.hide-from-print
-  .navbar-inner
-    .container
-      %ul.nav
-        - navigation_links.each do |link|
-          %li{class: link.tab_class(controller), id: link.tab_id }
-            - if link.subnav.present?
-              %li.dropdown
-                = link_to "#", class: "dropdown-toggle", data: { toggle: :dropdown } do
-                  = link.text
-                  %b.caret
-                %ul.dropdown-menu
-                  - link.subnav.each do |subnav_link|
-                    %li= subnav_link.to_html
-            - else
-              = link.to_html
-
-      - if !acting_as? && current_user
-        = form_tag global_search_path, class: "navbar-search pull-right" do
-          = text_field_tag :search, nil, placeholder: t("global_search.search"), class: "search-query span2"
-
-      %ul.nav.pull-right
-        = render "shared/nav/manage_facilities"
-        - global_navigation_links.each do |link|
-          %li{class: link.tab_class(controller)}= link.to_html
+- unless session_user.nil?
```

Comment:
We prefer not not using double negatives 😉 . So `if session_user` or `if `session_user.present?`.